frontend/README.md

```diff
@@ -1,6 +1,7 @@
 # Code.org Frontend
 
-This directory contains the codebase powering Code.org's marketing site and UI component library packages.
+This directory mostly contains the codebase powering Code.org's marketing site, UI component library packages and
+everything related to these two.
```

Comment:
```suggestion
This directory contains the libraries that power the Code.org Marketing and Code.org Curriculum websites.
```

Reply:
Just to note, `apps` might end up living in this directory as well, and probably be renamed so we may want to keep this general for now.